code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1726,8 +1726,7 @@
 			if(show_message)
 				to_chat(M, "<span class='warning'>You feel a slight burning sensation...</span>")
 		else if(M.getBruteLoss())
-			M.adjustBruteLoss(-reac_volume)
-			M.adjustFireLoss(reac_volume)
+			M.adjustFireLoss(M.adjustBruteLoss(-reac_volume))
```

Comment:
You sure it doesn't need to be negative so it heals?
```suggestion
			M.adjustFireLoss(-M.adjustBruteLoss(-reac_volume))
```

Reply:
also are you sure that this fixes it at all, it just seems to return the same value that's put in, what it actually does isn't put anywhere

Reply:
> You sure it doesn't need to be negative so it heals?

the negative is already there? it goes on reac_volume, this one heals brute and deals burn
although if the adjust damage procs dont return the actual amount healed i'll be very annoyed